include/dfrn.php

```diff
@@ -1585,18 +1716,22 @@ private static function process_relocation($xpath, $relocation, $importer) {
 		$relocate["poll"] = $xpath->query("dfrn:poll/text()", $relocation)->item(0)->nodeValue;
 		$relocate["sitepubkey"] = $xpath->query("dfrn:sitepubkey/text()", $relocation)->item(0)->nodeValue;
 
-		if (($relocate["avatar"] == "") AND ($relocate["photo"] != ""))
+		if (($relocate["avatar"] == "") AND ($relocate["photo"] != "")) {
 			$relocate["avatar"] = $relocate["photo"];
+		}
 
-		if ($relocate["addr"] == "")
+		if ($relocate["addr"] == "") {
 			$relocate["addr"] = preg_replace("=(https?://)(.*)/profile/(.*)=ism", "$3@$2", $relocate["url"]);
+		}
 
 		// update contact
 		$r = q("SELECT `photo`, `url` FROM `contact` WHERE `id` = %d AND `uid` = %d;",
 			intval($importer["id"]),
 			intval($importer["importer_uid"]));
-		if (!$r)
-			return false;
+
+		if (!dbm::is_result($r)) {
+			killme();
```

Comment:
reverted back and left my code commented-out only. Of course, the dbm::is_result() remains in ... ;-)